checklists/configure-php-cs-fix.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Configure PHP CS Fixer
+category: Back-End
+date: "2020-03-23"
+tags: ['php', 'symfony', 'configure']
+description: Sometimes, we mess up the basic rules of code style. So we need a friend to cover our back! With PHP we are using PHP CS Fixer and to configure it we follow the steps below.
```

Comment:
there should either "by" or "and" before the sentence "we follow the steps below." If these steps are necessary for configuring the PHP CS Fixer then you should add "by" otherwise "and" will be enough.

```suggestion
description: Sometimes, we mess up the basic rules of code style. So we need a friend to cover our back! With PHP we are using PHP CS Fixer and to configure it we follow the steps below.
```

Reply:
I have fixed this spelling thanks to you @akemalakcay 🎸